lib/actions/ui.js

```diff
@@ -230,3 +234,31 @@ export const MobileScreens = {
   SET_DATETIME: 7,
   RESULTS_SUMMARY: 8
 }
+
+export const ItineraryView = {
+  FULL: 'full',
+  HIDDEN: 'hidden',
+  SPLIT: 'split'
+}
```

Comment:
Please add JSDoc explaining what these represent.

Reply:
I added the JSDoc in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/794aeafcdf59f7ea394a00aac565daa4e7a9b2f0.